api/src/main/java/org/pmiops/workbench/db/model/DbWorkspace.java

```diff
@@ -531,6 +531,11 @@ public void setWorkspaceActiveStatusEnum(WorkspaceActiveStatus activeStatus) {
     setActiveStatus(DbStorageEnums.workspaceActiveStatusToStorage(activeStatus));
   }
 
+  @Transient
+  public boolean isActive() {
+    return getWorkspaceActiveStatusEnum().equals(WorkspaceActiveStatus.ACTIVE) ? true : false;
```

Comment:
Ternary is redundant here, i.e. removing ` ? true : false` is equivalent

also nit: safer in general to write `WorkspaceActiveStatus.ACTIVE.equals(x)` for null safety